mkdocs.yml

```diff
@@ -25,7 +25,24 @@ nav:
   - Changelog: "changelog.md"
 
 theme:
-  name: "material"
+  name: material
+  features:
+    - navigation.top
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: indigo
+      accent: teal
```

Comment:
I selected colors that wouldn't change too much compared to the current ones. Feel free to pick others: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-palette

Reply:
It seems to render dark links in a hard to read color, can you fix that please? 

![Screenshot_20220109-074800_Firefox](https://user-images.githubusercontent.com/973709/148672539-dd3beed0-ac30-4d2f-abcc-aa8e6e3860e1.jpg)